mirrord/agent/src/steal/api.rs

```diff
@@ -65,6 +79,7 @@ impl TcpStealerApi {
             close_permit: Some(close_permit),
             daemon_rx,
             task_status,
+            response_body_txs: HashMap::new(),
```

Comment:
Sender has to be cleaned up when the `TcpConnectionStealer` sends `DaemonTcp::Close` for the whole connection. You can detect this case in `TcpStealerApi::recv`